src/main/resources/assets/computercraft/lang/de_DE.lang

```diff
@@ -11,17 +11,17 @@ tile.computercraft:command_computer.name=Befehlscomputer
 tile.computercraft:advanced_modem.name=Ender-Modem
 tile.computercraft:speaker.name=Lautsprecher
 
-tile.computercraft:turtle.name=Roboter
-tile.computercraft:turtle.upgraded.name=%s-Bot
-tile.computercraft:turtle.upgraded_twice.name=%s & %s-Bot
-tile.computercraft:advanced_turtle.name=Erweiterter Bot
-tile.computercraft:advanced_turtle.upgraded.name=Erweiterter %s-Bot
-tile.computercraft:advanced_turtle.upgraded_twice.name=Erweiterter %s & %s-Bot
+tile.computercraft:turtle.name=Turtle
+tile.computercraft:turtle.upgraded.name=%s Turle
+tile.computercraft:turtle.upgraded_twice.name=%s & %s Turtle
+tile.computercraft:advanced_turtle.name=Erweitertes Turtle
+tile.computercraft:advanced_turtle.upgraded.name=Erweiterter %s Turtle
```

Comment:
Inconsistent genus here compared to line 17. Since it's a loan word, it's hard to choose one, but I'd personally go with feminine.

Reply:
One other option would be `Turtle (%s)`. That's always a viable fallback, despite not being THAT nice.